functions/go/set-namespace/go.mod

```diff
@@ -1,5 +1,12 @@
-module github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/set-namespace
+module main
 
 go 1.14
 
-require sigs.k8s.io/kustomize/kyaml v0.6.1-0.20200819135012-e6ee03e3e3e0
+require (
+	github.com/spf13/cobra v1.0.0
+	sigs.k8s.io/kustomize/api v0.6.4
+	sigs.k8s.io/kustomize/kyaml v0.9.4
+	sigs.k8s.io/yaml v1.2.0
+)
+
+replace sigs.k8s.io/kustomize/api v0.6.4 => sigs.k8s.io/kustomize/api v0.0.0-20201120230733-052a6b4e967b
```

Comment:
I'm a little confused here. If 0.6.4 is a released version, we don't need this line.

Reply:
We need some new changes in api module and it hasn't released. We can wait for api 0.6.5. It should be released this week or next week.

Reply:
Please add a TODO comment here to remind us to do that.